benchmarks/progress.py

```diff
@@ -67,3 +67,24 @@ def get_progress_since(date):
                 result.append(progress)
 
     return result
+
+
+def get_progress_betwewn_results(current, baseline):
```

Comment:
there is a typo in the function name
currently: betwewn
probably should be: between

It is consistent in the call in tasks.py, so there is no problem right now :)